src/vstt/display_widget.py

```diff
@@ -21,20 +23,43 @@ def __init__(
         self._experiment: Experiment = Experiment()
         self._widgets: dict[str, QtWidgets.QCheckBox] = {}
 
+        # Main layout
         outer_layout = QtWidgets.QVBoxLayout()
-        group_box = QtWidgets.QGroupBox("Display Options")
-        scroll_area = QtWidgets.QScrollArea()
-        scroll_area.setWidgetResizable(True)
-        scroll_area.setWidget(group_box)
-        outer_layout.addWidget(scroll_area)
-        inner_layout = QtWidgets.QVBoxLayout()
-        group_box.setLayout(inner_layout)
+
+        # Tree widget for collapsible categories
+        tree_widget = QtWidgets.QTreeWidget()
+        tree_widget.setHeaderHidden(True)  # Hide header
+        outer_layout.addWidget(tree_widget)
+
+        # Labels for display options
         labels = display_options_labels()
-        for _row_index, key in enumerate(labels.keys()):
-            checkbox = QtWidgets.QCheckBox(f"{labels[key]}", self)
-            inner_layout.addWidget(checkbox)
-            checkbox.clicked.connect(self._update_value_callback(key))
-            self._widgets[key] = checkbox
+        collapsible_labels = display_options_groups()
+        collapsible_keys = []
+
+        # Add collapsible groups
+        for group_title, keys in collapsible_labels.items():
+            parent_item = QtWidgets.QTreeWidgetItem(tree_widget)
+            parent_item.setText(0, group_title)
+            parent_item.setFont(0, QFont("Arial", 11, QFont.Bold))
+
+            for key in keys:
+                collapsible_keys.append(key)
+                if key in labels:
+                    child_item = QtWidgets.QTreeWidgetItem(parent_item)
+                    checkbox = QtWidgets.QCheckBox(f"{labels[key]}")
+                    tree_widget.setItemWidget(child_item, 0, checkbox)
+                    checkbox.clicked.connect(self._update_value_callback(key))
+                    self._widgets[key] = checkbox
+
+        # Add non-collapsible options
+        for key, label in labels.items():
+            if key not in collapsible_keys:
+                item = QtWidgets.QTreeWidgetItem(tree_widget)
+                checkbox = QtWidgets.QCheckBox(f"{label}")
+                tree_widget.setItemWidget(item, 0, checkbox)
+                checkbox.clicked.connect(self._update_value_callback(key))
+                self._widgets[key] = checkbox
+
         self.setLayout(outer_layout)
```

Comment:
see https://github.com/ssciwr/vstt/blob/main/tests/test_display_widget.py for the existing tests of this widget as an example of how to test these gui elements

Note these tests still pass even though you changed this widget, as they iterate over the checkboxes in the display so are not affected by the checkboxes now being inside a treewidget, but maybe you could add something there to check that the tree widget has the expected groups?